action/protocol/staking/validations.go

```diff
@@ -17,7 +17,6 @@ import (
 // Errors
 var (
 	ErrInvalidAmount       = errors.New("invalid staking amount")
```

Comment:
also replace this with `action.ErrInvalidAmount`

Reply:
done